quic/s2n-quic-core/src/connection/limits.rs

```diff
@@ -52,14 +53,14 @@ pub struct Limits {
     pub(crate) bidirectional_remote_data_window: InitialMaxStreamDataBidiRemote,
     pub(crate) unidirectional_data_window: InitialMaxStreamDataUni,
     pub(crate) max_open_bidirectional_streams: InitialMaxStreamsBidi,
-    pub(crate) max_open_local_unidirectional_streams: InitialMaxStreamsUni,
+    pub(crate) max_open_local_unidirectional_streams: LocalUniDirectionalLimit,
```

Comment:
You lost "Streams" in the new name, maybe call it `LocalUniDirectionalStreamLimit`, etc

Reply:
its part of the streams module.. actually I should rename it to `LocalUniDirectional`
`stream::limits::{LocalMaxSendBufferSize, LocalUniDirectionalLimit}`

Reply:
maybe use the full path then:
 ```rust
pub(crate) max_open_local_unidirectional_streams: stream::limits::LocalUniDirectional,
```

Reply:
and from https://github.com/aws/s2n-quic/pull/1394/files#r919230695, it should be `LocalUnidirectional`